elements/pfe-card/src/pfe-card.scss

```
@@ -162,8 +163,16 @@ $size-small: (
   &__header {
     z-index: 2;
     // Declare the header background color
-    background-color: #{pfe-local(BackgroundColor, $region: header)};
-    color:            #{pfe-local(Color, $region: header)};
+    background-color: #{pfe-local($cssvar: BackgroundColor, $region: header)};
```

```suggestion
    background-color: pfe-local(BackgroundColor, $region: header);
```

Reminder that you don't need to interpolate unless you're assigning the function to a custom property.  Also, we'd discussed previously dropping the `$cssvar` for readability.

---

resolved